examples/onprem_demo/variables.tf

```
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// The ID of the Onprem GCP project that is going to be created
+variable "onprem_project_id" {}
+
+// The ID of the VPC Service Control project that is going to be created
+variable "vpc_sc_project_id" {}
+
+// Path to the service account .json file
+variable "credentials_path" {}
+
+// Organization ID, which can be found at `gcloud organizations list`
+variable "organization_id" {}
+
+// Billing account ID to which the new project should be associated
+variable "billing_account_id" {}
+
+// GCP Region (like us-west1, us-central1, etc)
+variable "region" {}
```

Per our standards to make examples easier to run we add default variables where it's possible.
Please add to all other region variables as well in other variables.tf files.
```suggestion
variable "region" {
  description = "GCP Region"
  default = "us-central1"
}

```

---

Done.